File: proj.android_studio/ee-x-store/src/main/java/com/ee/store/PurchasesUpdate.java
package com.ee.store;

import androidx.annotation.Nullable;

import com.android.billingclient.api.BillingClient.BillingResponseCode;
import com.android.billingclient.api.Purchase;

import java.util.List;

final class PurchasesUpdate {
    @BillingResponseCode
    public final int code;

    @Nullable
    public final List<Purchase> purchases;

    PurchasesUpdate(@BillingResponseCode int code, @Nullable List<Purchase> purchases) {
        this.code = code;
        this.purchases = purchases;
    }
}

File: proj.android_studio/ee-x-store/src/main/java/com/ee/store/PurchasesUpdate.kt
package com.ee.store

import com.android.billingclient.api.BillingClient.BillingResponseCode
import com.android.billingclient.api.Purchase

internal class PurchasesUpdate(
    @field:BillingResponseCode @param:BillingResponseCode val code: Int,
    val purchases: List<Purchase>)